DoSomething/gladiator
# Remove duration, set end date on split

#### What's this PR do?
* Removed the duration field on contests and all references to the duration field. 
* Add field to split form that admins can use to set the end date for competitions when they do a split. Also added all the validation, split logic updates to accommodate this. 
* Updated the split form view a bit. It didn't seem useful to list all of the users in each competition at this point in the flow, but I can either add this back in later if folks really need it, or we can maybe link out to another view that has a list of all the users. 

The new split view looks like this:

<img width="913" alt="screen shot 2016-03-23 at 2 43 08 pm" src="https://cloud.githubusercontent.com/assets/1700409/13996790/9c60c032-f105-11e5-8111-d7d0567bcc17.png">

![serena-split](https://cloud.githubusercontent.com/assets/1700409/13996526/8f619d4e-f104-11e5-9b9a-da3f4a3e9f05.gif)


#### How should this be manually tested?
Pull down this branch 
Run `composer update` to get the `doctrine/dbal` package
Run `php artisan:migrate` to do the db updates
Check that duration field is gone-zo.

Then go to split a waiting room, 
you should be able to set a date that is after today as the end date for the competitions.

#### Any background context you want to provide?

NOTE: The `doctrine/dbal` package is [required in order to do column drops](https://laravel.com/docs/5.2/migrations#dropping-columns).
 
![674702c838b](https://cloud.githubusercontent.com/assets/1700409/13996609/f71e3f32-f104-11e5-8938-9e19432b9374.gif)

#### What are the relevant tickets?
Fixes #94 

@angaither @weerd @deadlybutter
